src/documentation/print-faq-wiki.ts

```diff
@@ -0,0 +1,76 @@
+import { setMinLevelOfAllLogs } from '../../test/functionality/_helper/log';
+import { LogLevel } from '../util/log';
+import { autoGenHeader } from './doc-util/doc-auto-gen';
+import { FlowrWikiBaseRef, getFilePathMd } from './doc-util/doc-files';
+import { codeBlock } from './doc-util/doc-code';
+
+function print(): string {
+	return `${autoGenHeader({ filename: module.filename, purpose: 'frequently asked questions' })}
```

Comment:
I think in the future this should be a flexible render and the code should only hold `registerAnswer`... or something similar to the server messages. But this may be a future iusse :)

Reply:
#1607